index.ios.js

```diff
@@ -4,6 +4,15 @@ var React = require('react-native');
 var { NativeModules, requireNativeComponent } = React;
 
 var MapMixins = {
+  addPackForRegion(mapRef, options) {
+    NativeModules.MapboxGLManager.addPackForRegion(React.findNodeHandle(this.refs[mapRef]), options);
+  },
+  getPacksWithCompletionHandler(mapRef, callback) {
```

Comment:
There's no longer any reason to make this asynchronous. The underlying property is completely synchronous now.